android/app/src/main/res/layout/content_session_detail.xml

```diff
@@ -33,133 +33,167 @@
                     android:layout_height="wrap_content"
                     android:layout_marginBottom="@dimen/layout_margin_medium"
                     android:textColor="@color/primary_text" />
-
-                <RelativeLayout
+                <LinearLayout
```

Comment:
I'll suggest using a GridLayout here instead.
Will reduce the XML nesting a lot.

Reply:
@the-dagger okay!